docs/3.x.x/en/configurations/configurations.md

```diff
@@ -179,6 +179,9 @@ Most of the application's configurations are defined by environment. It means th
        - `password` (string): Password used to establish the connection.
        - `options` (object): List of additional options used by the connector.
        - `timezone` (string): Set the default behavior for local time (used only for a SQL database). Default value: `utc`.
+    - `options` Options used for database connection.
```

Comment:
Not directly related to this PR but the other options should also be added as an example such as `debug` and (although a rare usecase) `acquireConnectionTimeout`